### ISSUE TITLE
Unused variables in melt.m

### ISSUE THREAD

**Issue by chadagreene** [alex-s-gardner/GEMB#3]
@alex-s-gardner There are a handful of variables in `melt.m` that never get used (e.g., [`surplusT`](https://github.com/alex-s-gardner/GEMB/blob/eed229e46ab77f5cadf7f8b6737c74594a7a9880/GEMB/melt.m#L61)). Do I have your blessing to delete them?

**Comment by alex-s-gardner** [comment]
I agree, we should delete all occurrences of unused variables that are likely introduces during development

**Comment by NJSchlegel** [comment]
Isn't it used on line 130 to throw a warning?

**Comment by chadagreene** [comment]
@NJSchlegel Nope, `surplusT` is never actually used. Line 130 just says:
```matlab 
surplusT=max(0, exsT(i) - LF/CI);
surplusE=surpE(i);
display([' WARNING: surplus energy at the base of GEMB column' newline])
```

**Comment by NJSchlegel** [comment]
Got it, the value is actually stored instead in surpT in this loop.  Probably was used for printing the values in the warning statement and some point.
